src/app/utils/githubUtils.ts

```diff
@@ -0,0 +1,92 @@
+import {Config} from '../components/СonfigForm';
+import {Octokit} from '@octokit/core';
+import { Endpoints } from "@octokit/types";
+
+type listUserReposResponse = Endpoints["GET /repos/{owner}/{repo}"]["response"];
+
+export type File = {
+  path: string;
+  content: string;
+};
+
+type gitType = 'tree' | 'blob' | 'commit';
+type commitMode = '100644' | '100755' | '040000' | '160000' | '120000';
+
+const COMMIT_MESSAGE = 'Updating design system styles';
+// const PULL_REQUEST_TITLE = 'Updating design system styles';
+
+type MultipleFilesCommitter = (options: Config, files: File[]) => Promise<void>;
+export const commitMultipleFiles: MultipleFilesCommitter = async (
+  options,
```

Comment:
Тут кстати можно было вот так написать, тогда константы снизу вообще не нужны
```suggestion
  options: {ownerName: owner, repoName: repo, headBranch: ref, token, },
```